### PR TITLE
Remove guts config

### PR DESCRIPTION
**Pull request by balagopalraj**

### CODE REVIEW

File: devstack/settings [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # -------------------------------------
 # Set up default repos
 GUTS_REPO="https://github.com/aptira/guts.git"
-GUTS_BRANCH=${GUTS_BRANCH:-master}
+GUTS_BRANCH=${GUTS_BRANCH:-mysql-backend}
```
Comment (BharatKumarK) [inline devstack/settings]: I think we should set GUTS_BRANCH to master, which has latest code.

File: devstack/settings [hunk 2]
```diff
@@ -19,6 +19,18 @@ GUTS_ENABLE_MODEL_POLICY_ENFORCEMENT=${GUTS_ENABLE_MODEL_POLICY_ENFORCEMENT:-Fal
...
+# Set up default repos
+GUTS_DASHBOARD_REPO="https://github.com/aptira/guts-dashboard.git"
+GUTS_DASHBOARD_BRANCH=${GUTS_DASHBOARD_BRANCH:-master}
+
+# Set up default directories
+GUTS_DASHBOARD_DIR=$DEST/guts-dashboard
+HORIZON_DIR=$DEST/horizon
+GUTS_DASHBOARD_CACHE_DIR=${GUTS_DASHBOARD_CACHE_DIR:-/tmp/guts}
+
+HORIZON_SETTINGS=${HORIZON_SETTINGS:-$HORIZON_DIR/openstack_dashboard/settings.py}
```
Comment (BharatKumarK) [inline devstack/settings]: Whare are you using this "HORIZON_SETTINGS" variable?

File: devstack/plugin.sh [hunk 3]
```diff
@@ -207,125 +208,45 @@ function cleanup_guts() {
...
-#GUTS_CONFIG_SECTION_END
 
-EOF
-
-    cat "$horizon_config_part" >> "$HORIZON_LOCAL_CONFIG"
-
-    if [[ -f "$HORIZON_LOCAL_CONFIG" ]]; then
-        sed -e "s/\(^\s*OPENSTACK_HOST\s*=\).*$/\1 '$HOST_IP'/" -i "$HORIZON_LOCAL_CONFIG"
+    if [[ -f "$HORIZON_LOCAL_SETTINGS" ]]; then
+        sed -e "s/\(^\s*OPENSTACK_HOST\s*=\).*$/\1 '$HOST_IP'/" -i "$HORIZON_LOCAL_SETTINGS"
```
Comment (BharatKumarK) [inline devstack/plugin.sh]: Could you please explain what this statement do?

**Comment by balagopalraj** [comment]
I will reopen this.

**Pull request closed without merging**